content/en/actions/agents/_index.md

```diff
@@ -0,0 +1,154 @@
+---
+title: Agents
+description: Build custom AI agents that can access the Action Catalog. 
+disable_toc: false
+further_reading:
+- link: "/actions/actions_catalog/"
+  tag: "Documentation"
+  text: "Action Catalog"
+- link: "https://www.datadoghq.com/knowledge-center/aiops/ai-agents/"
+  tag: "Knowledge Center"
+  text: "What are AI agents and how do they work?"
+---
+
+{{< callout url="https://app.datadoghq.com/actions/agents" btn_hidden="false" header="Join the Preview!" >}}
+Agents is in Preview. Click <b>Request Access</b> and fill in the Datadog Product Preview Program form.
```

Comment:
I still think a plural verb reads more naturally, or maybe "The Agents product is in Preview" if you want to keep singular.

```suggestion
Agents are in Preview. Click <b>Request Access</b> and fill in the Datadog Product Preview Program form.
```